lib/client.js

```diff
@@ -355,8 +356,8 @@ Client.prototype.createTopics = function (topics, isAsync, cb) {
     cb = isAsync;
     isAsync = true;
   }
+  validateKafkaTopics(topics);
```

Comment:
I used Error() object with throw keyword in the `validateKafkaTopics ` topics function. So even though isAsync is true, error will be thrown.